src/pages/StockConfiguration/UseCaseConfiguration/SelectProgram.js

```diff
@@ -1,14 +1,20 @@
 import i18n from '@dhis2/d2-i18n'
+import find from 'lodash/find'
+import reject from 'lodash/reject'
 import PropTypes from 'prop-types'
 import React from 'react'
-import { Select } from '../../../components'
-import { defaultStockCase } from '../helper'
+import { Input, Select } from '../../../components'
+import { defaultStockCase, getElementName } from '../helper'
 import { useLogisticPrograms } from '../logisticHelper'
 
 const program = 'programUid'
 
-const SelectProgram = ({ settings, handleSettings }) => {
+const SelectProgram = ({ settings, handleSettings, useCases, disabled }) => {
     const { programs } = useLogisticPrograms()
+    const programList = programs
```

Comment:
Hey,
I am curious to know, how does the user understand why the `programList` dropdown is empty? 
Does it make sense to disable the `Add program` button and add a hover tooltip explaining that all the configurations are already in use and you need to delete/edit an item? Or maybe explain it near the `programList` dropdown when it's empty?

<img width="1315" alt="Screenshot 2023-03-22 at 15 42 25" src="https://user-images.githubusercontent.com/6817073/226941659-4df502d2-5f7c-43ab-bf3d-274670b03b63.png">

Thank you!

Reply:
Hi, thanks for the suggestions. 
I'm adding a tooltip and disabling the add button.